### PR TITLE
doc(release): create v4.6.0

### PR DESCRIPTION
**Pull request by floreal15**
https://github.com/tracim/tracim/issues/6215

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- ~~[ ] Manual, quality tests have been done~~ No test necessary here

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 4.6.0 / 2023-07-26
+
+### ✨ New Features
+
+- Add command to really delete data in tracimcli
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:7]:
```suggestion
- Add command to really delete data with tracimcli command
```
Something like that it is not better?

Do you have an other idea @Quiwy @lebouquetin

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 4.6.0 / 2023-07-26
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:3]: the date is the date when release is available. So today (27) or tomorrow (28).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+
+### ✨ New Features
+
+- Add command to really delete data in tracimcli
+- Increase size of emojis in TinyMCE
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:8]: This is a really new feature or a bug fix ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
+- Upgrade pre-commit version to flacke8 6.0.0, trailing-whitespace v4.4.0, black 23.3.0, isort v5.10.1 and seed-isort-config v2.2.0
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:9]: I think this is not really a new feature. This is more a fixed issues, no ?

And I think it is better to just inform that we have upgraded some dependencies (and maybe add name of this dependencies)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,21 @@
...
+
+### 🐛 Fixed Issues
+
+- Frontend: [#4591] https://github.com/tracim/tracim/issues/4591
+
+- Backend: [#6201] https://github.com/tracim/tracim/issues/6201
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:15]: I think syntax is not good. In older version we use `[#number](url)`

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,21 @@
...
+- Increase size of emojis in TinyMCE
+- Upgrade pre-commit version to flacke8 6.0.0, trailing-whitespace v4.4.0, black 23.3.0, isort v5.10.1 and seed-isort-config v2.2.0
+
+### 🐛 Fixed Issues
+
+- Frontend: [#4591] https://github.com/tracim/tracim/issues/4591
+
+- Backend: [#6201] https://github.com/tracim/tracim/issues/6201 
+
+### 💥 Breaking/Important changes
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:17]: If we not have information here, we can delete this lines.

**Comment by PhilippeAccorsi** [comment]
Problem of timeout in python test.
I restart backend test.

File: CHANGELOG.md (hunk 3 above)
Comment (floreal15) [reply CHANGELOG.md:8]: It's not really a bug fix because it wasn't a bug, it's just an ux change

**Comment by PhilippeAccorsi** [comment]
@floreal15 do you know if documentation exists for the new feature with tracimcli command ?

**Comment by floreal15** [comment]
> @floreal15 do you know if documentation exists for the new feature with tracimcli command ?

yes in doc/backend/cli.md

File: CHANGELOG.md (hunk 4 above)
Comment (floreal15) [reply CHANGELOG.md]: I placed it in fixed issues

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,20 +1,18 @@
...
 ### 🐛 Fixed Issues
 
-- Frontend: [#4591] https://github.com/tracim/tracim/issues/4591
+- Frontend: [#4591] (https://github.com/tracim/tracim/issues/4591)
 
-- Backend: [#6201] https://github.com/tracim/tracim/issues/6201 
-
-### 💥 Breaking/Important changes
+- Backend: [#6190] (https://github.com/tracim/tracim/issues/6190),
+  [#6201] (https://github.com/tracim/tracim/issues/6201)
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md]: I'm not sure space working between `] (`

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,19 @@
...
+
+- Add command to really delete data with tracimcli command
+- Increase size of emojis in TinyMCE
+
+### 🐛 Fixed Issues
+
+- Frontend: [#4591](https://github.com/tracim/tracim/issues/4591)
+
+- Backend: [#6190](https://github.com/tracim/tracim/issues/6190),
+  [#6201] (https://github.com/tracim/tracim/issues/6201)
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:15]:
```suggestion
  [#6201](https://github.com/tracim/tracim/issues/6201)
```

**Review by PhilippeAccorsi — COMMENTED** [review]
J'avais pas pensé, mais tu peux je pense te baser sur la structuration de la 4.5.0 si tu veux pour mettre en avant les nouveautés :)

Pour le reste c'est good, j'ai mis une suggestion pour l'espace en trop.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,19 @@
...
+
+### ✨ New Features
+
+- Add command to really delete data with tracimcli command
+- Increase size of emojis in TinyMCE
+
+### 🐛 Fixed Issues
+
+- Frontend: [#4591](https://github.com/tracim/tracim/issues/4591)
+
```
Comment (PhilippeAccorsi) [inline CHANGELOG.md:13]: Just remove this empty line and it is ok

**Review by PhilippeAccorsi — APPROVED** [review]
Just remove one empty line and we can merge in develop (with squash commit)
After that, we need to create MR to merge develop in master (waiting test and merge without squash).

**Pull request merged**